lib/PuppeteerSharp/Page.cs

```diff
@@ -535,6 +535,15 @@ public Task<T> EvaluateFunctionAsync<T>(string script, params object[] args)
             return navigationTask.Result;
         }
 
+        public Task WaitForTimeoutAsync(int milliseconds)
+            => MainFrame.WaitForTimeoutAsync(milliseconds);
+
+        public Task WaitForFunctionAsync(string script, WaitForFunctionOptions options = null, params object[] args)
```

Comment:
Can we add this override?
`public Task WaitForFunctionAsync(string script, params object[] args)`

or the `params object[]` gets in the middle?